### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by nxaden**
No comment....

### CODE REVIEW

File: src/main/java/Cat.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    @Override
+    public int getPrice()
+    {
+        return 1000;
+    }
+
+    @Override
+    /*
+    This method returns an int
```
Comment (sourenrex) [inline src/main/java/Cat.java]: should say this method returns a string not an int

File: src/test/java/CatTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import static org.junit.Assert.assertEquals;
+
+public class CatTest {
+     Cat c;
+
+    @Before
+    public void setUp() throws Exception {
+        c = new Cat();
+    }
+
```
Comment (sourenrex) [inline src/test/java/CatTest.java:13]: didnt add test case get price

Comment (abdullah-sha256) [reply src/test/java/CatTest.java:13]: ohhh yeaaa..

**Review by sourenrex — APPROVED** [review]
Good job

**Review by sourenrex — APPROVED** [review]

**Pull request merged**